frontend/src/container/NewDashboard/DashboardDescription/index.tsx

```diff
@@ -63,6 +67,30 @@ interface DashboardDescriptionProps {
 	handle: FullScreenHandle;
 }
 
+function removeValuesFromVariables(
+	selectedData: DashboardData,
+): Omit<DashboardData, 'uuid'> {
+	if (!selectedData?.variables) {
+		const { uuid, ...rest } = selectedData;
+		return rest;
```

Comment:
since we are removing uuid while exporting the dashboard we should add it back automatically while importing

Reply:
https://github.com/SigNoz/signoz/pull/6145#discussion_r1799460569

Reply:
Done, silently removed uuid when empty